### ISSUE TITLE
Feature request: load random values from a sample json

### ISSUE THREAD

**Issue by shahimclt** [jrebocho/vscode-random#42]
I know this is a long shot, but it would be nice to be able to insert random values from a json data source.

Let me elaborate: 
I have a json with almost 2000 entries worth of randomized data that goes like this (some data omitted for brevity):

    [{
        "sl": "142",
        "emp": { "i": "", "empid": "49423", "nm": "Devakumar\u00a0 Jonnalagadda", "pn": "Manager, Finance", "clr": "cyan" },
        "bk": { "l": 5, "nm": "Receivables - FIN", "clr": "cyan" },
        "pr": { "d": 9, "t": 39 },
        "act": { "ex": true }
    },
    {
        "sl": "143",
        "emp": { "i": "", "empid": "67298", "nm": "Venkatesan\u00a0 Jagadeesan ", "pn": "Senior Manager, Finance", "clr": "cyan" },
        "bk": { "l": 35, "nm": "Payables - FIN", "clr": "cyan" },
        "pr": { "d": 21, "t": 34 },
        "act": { "ex": true }
    },
    {
        "sl": "134",
        "emp": { "i": "", "empid": "54087", "nm": "Rupesh\u00a0 Thomas ", "pn": "Manager, Investment", "clr": "blue" },
        "bk": { "l": 3, "nm": "Investments - FIN", "clr": "blue" },
        "pr": { "d": 9, "t": 50 },
        "act": { "ex": true }
    }]

In fact, I have multiple files like this with different list from which I would like to pull random data.
It would be super helpful to be able to specify a file like `dummyData.json` and then the field like `emp.pn` and be able to pull random data from all the `pn` fields.

Or, We could simply be able to generate random strings from a sample input which is a json array of strings, without complicating things too much.

**Comment by jrebocho** [comment]
Hi @shahimclt ,

Thanks for the suggestion! Seems like a cool feature to have.

I'll take this in consideration for future releases.

**Comment by Ardianii1** [comment]
> Hi @shahimclt ,
> 
> Thanks for the suggestion! Seems like a cool feature to have.
> 
> I'll take this in consideration for future releases.

Yeah with this feature it completes the extension , I'm waiting for this feature too
